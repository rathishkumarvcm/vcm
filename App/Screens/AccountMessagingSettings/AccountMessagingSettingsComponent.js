import React, { Component } from 'react';
import { Text, View, ScrollView, TouchableOpacity } from 'react-native';
import PropTypes from 'prop-types';
import { styles } from './styles';
import {
    GHeaderComponent,
    GIcon,
    GFooterSettingsComponent,
} from '../../CommonComponents';
import gblStrings from '../../Constants/GlobalStrings';

class AccountMessagingSettingsComponent extends Component {
    constructor(props) {
        super(props);
        //  set true to isLoading if data for this screen yet to be received and wanted to show loader.
        this.state = {
            isLoading: false,
        };
    }   

    componentDidMount() { }

    navigateaccountMessagingGoals = () =>
        this.props.navigation.navigate('accountMessagingGoals');

    navigateaccountMessagingAdvice = () =>
        this.props.navigation.navigate('accountMessagingAdvice');

    navigateaccountMessagingGeneralDocuments = () =>
        this.props.navigation.navigate('accountMessagingGeneralDocuments');

    navigateAccountMessagingSecurityAndFraud = () =>
        this.props.navigation.navigate('AccountMessagingSecurityAndFraud');

    navigateaccountMessagingDeviceManagement = () =>
        this.props.navigation.navigate('accountMessagingDeviceManagement');   

    goBack = () => {
        this.props.navigation.goBack();
    };

    render() {
        return (
            <View style={styles.container}>
                <GHeaderComponent navigation={this.props.navigation} />
                <ScrollView style={styles.scrollViewFlex}>
                    <View style={styles.settingsView}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.goBack}
                        >
                            <Text style={styles.settingsInfo}>
                                {gblStrings.settingAccountMessaging.setting}
                            </Text>
                        </TouchableOpacity>
                        <Text style={styles.settingsInfoCurrent}>
                            {gblStrings.settingAccountMessaging.accountMessagingArrow}
                        </Text>
                    </View>
                    <View style={styles.settingsInfoHead}>
                        <Text style={styles.settingsInfoHeadTilte}>
                            {gblStrings.settingAccountMessaging.accountMessagingTitle}
                        </Text>
                        <Text style={styles.settingsInfo}>
                            {gblStrings.settingAccountMessaging.accountMessagingTitleDesc}
                        </Text>
                    </View>

                    <View style={styles.listContainer}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.navigateaccountMessagingGoals}
                        >
                            <View style={styles.listContainerItem}>
                                <GIcon
                                    name="appstore1"
                                    type="antdesign"
                                    size={35}
                                    color="#000000"
                                />
                                <Text style={styles.listContainerItemTextTitle}>
                                    {gblStrings.settingAccountMessaging.goalTitle}
                                </Text>
                            </View>
                            <View>
                                <Text style={styles.listContainerItemTextSubtitle}>
                                    {gblStrings.settingAccountMessaging.goalTitleDesc}
                                </Text>
                            </View>
                        </TouchableOpacity>
                    </View>

                    <View style={styles.listContainer}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.navigateaccountMessagingAdvice}
                        >
                            <View style={styles.listContainerItem}>
                                <GIcon
                                    name="appstore1"
                                    type="antdesign"
                                    size={35}
                                    color="#000000"
                                />
                                <Text style={styles.listContainerItemTextTitle}>
                                    {gblStrings.settingAccountMessaging.adviceTitle}
                                </Text>
                            </View>
                            <View>
                                <Text style={styles.listContainerItemTextSubtitle}>
                                    {gblStrings.settingAccountMessaging.adviceTitleDesc}
                                </Text>
                            </View>
                        </TouchableOpacity>
                    </View>

                    <View style={styles.listContainer}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.navigateaccountMessagingGeneralDocuments}
                        >
                            <View style={styles.listContainerItem}>
                                <GIcon
                                    name="appstore1"
                                    type="antdesign"
                                    size={35}
                                    color="#000000"
                                />
                                <Text style={styles.listContainerItemTextTitle}>
                                    {gblStrings.settingAccountMessaging.generalDocumentsTitle}
                                </Text>
                            </View>
                            <View>
                                <Text style={styles.listContainerItemTextSubtitle}>
                                    {gblStrings.settingAccountMessaging.generalDocumentsTitleDesc}
                                </Text>
                            </View>
                        </TouchableOpacity>
                    </View>

                    <View style={styles.listContainer}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.navigateAccountMessagingSecurityAndFraud}
                        >
                            <View style={styles.listContainerItem}>
                                <GIcon
                                    name="appstore1"
                                    type="antdesign"
                                    size={35}
                                    color="#000000"
                                />
                                <Text style={styles.listContainerItemTextTitle}>
                                    {
                                        gblStrings.settingAccountMessaging
                                            .securityAndFraudAlertsTitle
                                    }
                                </Text>
                            </View>
                            <View>
                                <Text style={styles.listContainerItemTextSubtitle}>
                                    {
                                        gblStrings.settingAccountMessaging
                                            .securityAndFraudAlertsTitleDesc
                                    }
                                </Text>
                            </View>
                        </TouchableOpacity>
                    </View>

                    <View style={styles.listContainer}>
                        <TouchableOpacity
                            style={styles.touchOpacityPosition}
                            onPress={this.navigateaccountMessagingDeviceManagement}
                        >
                            <View style={styles.listContainerItem}>
                                <GIcon
                                    name="appstore1"
                                    type="antdesign"
                                    size={35}
                                    color="#000000"
                                />
                                <Text style={styles.listContainerItemTextTitle}>
                                    {gblStrings.settingAccountMessaging.deviceManagementTitle}
                                </Text>
                            </View>
                            <View>
                                <Text style={styles.listContainerItemTextSubtitle}>
                                    {gblStrings.settingAccountMessaging.deviceManagementTitleDesc}
                                </Text>
                            </View>
                        </TouchableOpacity>
                    </View>

                    <GFooterSettingsComponent />
                </ScrollView>
            </View>
        );
    }
}

AccountMessagingSettingsComponent.propTypes = {
    navigation: PropTypes.instanceOf(Object),
};

AccountMessagingSettingsComponent.defaultProps = {};

export default AccountMessagingSettingsComponent;
