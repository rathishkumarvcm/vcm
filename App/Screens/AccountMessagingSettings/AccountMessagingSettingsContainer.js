import { connect } from "react-redux";
import AccountMessagingSettingsComponent from './AccountMessagingSettingsComponent';
import { getLogin } from "../../Reducers";
import { loginActions } from "../../Actions";

/*----------------------
                                  Redux Methods
                                                             -------------------------- */

const mapStateToProps = (state /* , props */) => (
  {
    dashboardData: getLogin(state)
  }
);

const mapDispatchToProps = {
  ...loginActions
};

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(AccountMessagingSettingsComponent);