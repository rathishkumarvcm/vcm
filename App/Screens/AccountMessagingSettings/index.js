import AccountMessagingSettingsComponent from './AccountMessagingSettingsContainer';

export default AccountMessagingSettingsComponent;
