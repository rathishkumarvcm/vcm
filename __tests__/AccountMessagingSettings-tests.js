import React from 'react';
import renderer from 'react-test-renderer';
import {shallow} from 'enzyme';
import AccountMessagingSettings from '../App/Screens/AccountMessagingSettings/AccountMessagingSettingsComponent';

describe('Account Messaging Settings',()=>{

    //  Snap shot 
    it('Match Snapshot', () => {  
      const tree = renderer.create(<AccountMessagingSettings />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  
    //  Render using shallow - shallow not includes child component
    test('render',()=>{
      const component = shallow(<AccountMessagingSettings />);  
      component.render();
    });

});