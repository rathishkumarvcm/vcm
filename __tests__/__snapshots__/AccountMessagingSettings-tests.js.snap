// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Account Messaging Settings Match Snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#F7FAFF",
      "flex": 1,
      "width": "100%",
    }
  }
>
  <View
    accessible={true}
    clickable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "flex": 0.12,
        "flexDirection": "row",
        "opacity": 1,
        "paddingLeft": "4%",
        "paddingRight": "4%",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "height": 100.5,
          "opacity": 1,
          "width": "25%",
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../App/Images/logo.png",
          }
        }
        style={
          Object {
            "alignItems": "flex-start",
            "justifyContent": "flex-start",
            "width": "100%",
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "width": "12%",
        }
      }
    />
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "marginTop": 24.5,
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 40,
            "width": 56,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 40,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "EvilIcons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      style={
        Object {
          "width": "10%",
        }
      }
    />
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={true}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "height": 30,
            "width": 44,
          }
        }
      >
        <Text
          allowFontScaling={false}
          style={
            Array [
              Object {
                "color": "black",
                "fontSize": 30,
              },
              Object {
                "backgroundColor": "transparent",
                "marginHorizontal": 5,
              },
              Object {
                "fontFamily": "Ionicons",
                "fontStyle": "normal",
                "fontWeight": "normal",
              },
              Object {},
            ]
          }
          testID="iconIcon"
        >
          
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      clickable={false}
      isTVSelectable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "alignItems": "flex-end",
          "justifyContent": "center",
          "opacity": 1,
        }
      }
    >
      <RNGestureHandlerButton
        collapsable={false}
        onGestureEvent={[Function]}
        onGestureHandlerEvent={[Function]}
        onGestureHandlerStateChange={[Function]}
        onHandlerStateChange={[Function]}
        rippleColor={0}
        style={
          Array [
            Object {
              "overflow": "hidden",
            },
            undefined,
          ]
        }
      >
        <View
          accessible={true}
          style={
            Object {
              "alignItems": "flex-end",
              "justifyContent": "center",
              "marginEnd": 10,
              "opacity": 1,
            }
          }
        >
          <View
            style={
              Object {
                "height": 40,
                "width": 56,
              }
            }
          >
            <Text
              allowFontScaling={false}
              style={
                Array [
                  Object {
                    "color": "black",
                    "fontSize": 40,
                  },
                  Object {
                    "backgroundColor": "transparent",
                    "marginHorizontal": 5,
                  },
                  Object {
                    "fontFamily": "Material Icons",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
              testID="iconIcon"
            >
              
            </Text>
          </View>
        </View>
      </RNGestureHandlerButton>
    </View>
  </View>
  <RCTScrollView
    style={
      Object {
        "flex": 0.85,
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "flexDirection": "row",
            "marginTop": 29.5,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#B2B2B2",
                "fontSize": 21.5,
                "marginRight": "4%",
              }
            }
          >
            Setting
          </Text>
        </View>
        <Text
          style={
            Object {
              "color": "#707070",
              "fontSize": 23,
              "fontWeight": "bold",
            }
          }
        >
          &gt;    Account Messaging
        </Text>
      </View>
      <View
        style={
          Object {
            "marginTop": 29.5,
            "paddingLeft": "4%",
            "paddingRight": "4%",
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#56565A",
              "fontSize": 33,
              "fontWeight": "bold",
              "paddingBottom": 6.5,
            }
          }
        >
          Account Messaging
        </Text>
        <Text
          style={
            Object {
              "color": "#B2B2B2",
              "fontSize": 21.5,
              "marginRight": "4%",
            }
          }
        >
          Lorem Ipsum is dummy text of the typesetting industry.
        </Text>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderRadius": 26.5,
            "borderWidth": 1,
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 39.5,
            "width": "92%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginLeft": "3%",
                "marginRight": "2%",
                "marginTop": 39.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "marginLeft": "2%",
                  "textTransform": "uppercase",
                }
              }
            >
              Goals
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#B2B2B2",
                  "fontSize": 23,
                  "lineHeight": 33,
                  "marginBottom": 57.5,
                  "marginLeft": "4%",
                  "marginTop": 41,
                  "width": "90%",
                }
              }
            >
              Lorem Ipsum has been the industrys standard dummy text ever since the 1500s
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderRadius": 26.5,
            "borderWidth": 1,
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 39.5,
            "width": "92%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginLeft": "3%",
                "marginRight": "2%",
                "marginTop": 39.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "marginLeft": "2%",
                  "textTransform": "uppercase",
                }
              }
            >
              Advice
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#B2B2B2",
                  "fontSize": 23,
                  "lineHeight": 33,
                  "marginBottom": 57.5,
                  "marginLeft": "4%",
                  "marginTop": 41,
                  "width": "90%",
                }
              }
            >
              Lorem Ipsum has been the industrys standard dummy text ever since the 1500s
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderRadius": 26.5,
            "borderWidth": 1,
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 39.5,
            "width": "92%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginLeft": "3%",
                "marginRight": "2%",
                "marginTop": 39.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "marginLeft": "2%",
                  "textTransform": "uppercase",
                }
              }
            >
              Investment Accounts
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#B2B2B2",
                  "fontSize": 23,
                  "lineHeight": 33,
                  "marginBottom": 57.5,
                  "marginLeft": "4%",
                  "marginTop": 41,
                  "width": "90%",
                }
              }
            >
              Lorem Ipsum has been the industrys standard dummy text ever since the 1500s
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderRadius": 26.5,
            "borderWidth": 1,
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 39.5,
            "width": "92%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginLeft": "3%",
                "marginRight": "2%",
                "marginTop": 39.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "marginLeft": "2%",
                  "textTransform": "uppercase",
                }
              }
            >
              Security And Fraud Alerts
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#B2B2B2",
                  "fontSize": 23,
                  "lineHeight": 33,
                  "marginBottom": 57.5,
                  "marginLeft": "4%",
                  "marginTop": 41,
                  "width": "90%",
                }
              }
            >
              Lorem Ipsum has been the industrys standard dummy text ever since the 1500s
            </Text>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#D4D4D4",
            "borderRadius": 26.5,
            "borderWidth": 1,
            "marginLeft": "4%",
            "marginRight": "4%",
            "marginTop": 39.5,
            "width": "92%",
          }
        }
      >
        <View
          accessible={true}
          clickable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "relative",
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "marginLeft": "3%",
                "marginRight": "2%",
                "marginTop": 39.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#000000",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "marginLeft": "2%",
                  "textTransform": "uppercase",
                }
              }
            >
              Device Management
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#B2B2B2",
                  "fontSize": 23,
                  "lineHeight": 33,
                  "marginBottom": 57.5,
                  "marginLeft": "4%",
                  "marginTop": 41,
                  "width": "90%",
                }
              }
            >
              Lorem Ipsum has been the industrys standard dummy text ever since the 1500s
            </Text>
          </View>
        </View>
      </View>
      <View>
        <View
          style={
            Object {
              "borderColor": "#DEDEDF",
              "borderTopWidth": 1,
              "marginTop": 65.5,
            }
          }
        />
        <View
          style={
            Object {
              "backgroundColor": "#FFFFFF",
            }
          }
        >
          <View
            style={
              Object {
                "marginBottom": 41,
                "marginTop": 41,
                "paddingLeft": "4%",
                "paddingRight": "4%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 29.5,
                  "fontWeight": "bold",
                }
              }
            >
              Investing involves risk including loss of principal.
            </Text>
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "lineHeight": 33,
                  "marginTop": 24.5,
                }
              }
            >
              Victory Mutual Funds and USAA Mutual Funds are distributed by Victory Capital Advisers, Inc. (VCA). VictoryShares ETFs and VictoryShares USAA ETFs are distributed by Foreside Fund Services, LLC (Foreside). VCA and Foreside are members of FINRA and SIPC. Victory Capital Management Inc. (VCM) is the investment adviser to the Victory Mutual Funds, USAA Mutual Funds, VictoryShares ETFs and VictoryShares USAA ETFs. VCA and VCM are not affiliated with Foreside. USAA is not affiliated with Foreside, VCM, or VCA. USAA and the USAA logos are registered trademarks and the USAA Mutual Funds and USAA Investments logos are trademarks of United Services Automobile Association and are being used by Victory Capital and its affiliates under license. Victory Capital means Victory Capital Management Inc., the investment manager of the USAA 529 College Savings Plan (Plan). The Plan is distributed by Victory Capital Advisers, Inc., a broker dealer registered with FINRA and an affiliate of Victory Capital. Victory Capital and its affiliates are not affiliated with United Services Automobile Association or its affiliates. USAA and the USAA logo are registered trademarks and the USAA 529 College Savings Plan logo is a trademark of United Services Automobile Association and are being used by Victory Capital and its affiliates under license.
            </Text>
          </View>
          <View
            style={
              Object {
                "marginBottom": 41,
                "marginTop": 33,
                "paddingLeft": "4%",
                "paddingRight": "4%",
              }
            }
          >
            <Image
              resizeMode="contain"
              source={
                Object {
                  "testUri": "../../../App/Images/logo.png",
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "marginLeft": "4%",
                "marginTop": 33,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#56565A",
                  "fontSize": 26.5,
                  "lineHeight": 33,
                  "marginTop": 24.5,
                }
              }
            >
              Connect with Us
            </Text>
          </View>
          <View
            style={
              Object {
                "flexDirection": "row",
                "marginBottom": 33,
                "marginLeft": "2%",
                "marginTop": 16.5,
              }
            }
          >
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#56565A",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
            <View
              style={
                Object {
                  "height": 35,
                  "width": 50,
                }
              }
            >
              <Text
                allowFontScaling={false}
                style={
                  Array [
                    Object {
                      "color": "#56565A",
                      "fontSize": 35,
                    },
                    Object {
                      "backgroundColor": "transparent",
                      "marginHorizontal": 5,
                    },
                    Object {
                      "fontFamily": "anticon",
                      "fontStyle": "normal",
                      "fontWeight": "normal",
                    },
                    Object {},
                  ]
                }
                testID="iconIcon"
              >
                
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "flexDirection": "row",
                "height": 65.5,
                "justifyContent": "center",
                "marginLeft": "2%",
                "marginTop": 24.5,
                "width": "100%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#5D83AE",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "paddingLeft": "2%",
                  "width": "50%",
                }
              }
            >
              Privacy Policy
            </Text>
            <Text
              style={
                Object {
                  "color": "#5D83AE",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "paddingLeft": "2%",
                  "width": "50%",
                }
              }
            >
              Fund Documents
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "white",
                "flexDirection": "row",
                "height": 65.5,
                "justifyContent": "center",
                "marginLeft": "2%",
                "marginTop": 24.5,
                "width": "100%",
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#5D83AE",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "paddingLeft": "2%",
                  "width": "50%",
                }
              }
            >
              User Agreements
            </Text>
            <Text
              style={
                Object {
                  "color": "#5D83AE",
                  "fontSize": 26.5,
                  "fontWeight": "bold",
                  "paddingLeft": "2%",
                  "width": "50%",
                }
              }
            >
              Support
            </Text>
          </View>
          <View
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#56565A",
                "height": 98.5,
                "justifyContent": "center",
                "marginTop": 33,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#FFFFFF",
                }
              }
            >
              Copyright Victory Capital Management Inc. ©2020
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
